scripts/release_scripts/deploy.py

```diff
@@ -220,12 +221,16 @@ def update_and_check_indexes(app_name):
         'https://console.cloud.google.com/datastore/indexes'
         '?project=%s') % app_name
     gcloud_adapter.update_indexes(INDEX_YAML_PATH, app_name)
-    if not gcloud_adapter.check_all_indexes_are_serving(app_name):
-        common.open_new_tab_in_browser_if_possible(indexes_page_url)
-        raise Exception(
-            'Please wait for all indexes to serve, then run this '
-            'script again to complete the deployment. For details, '
-            'visit the indexes page. Exiting.')
+
+    # Sleep for a minute before checking that all indexes are serving.
```

Comment:
Probably want to add a message here saying "Waiting for indexes to server" or something like that? (before the wait)

Otherwise, might look like it's hanging.

Also -- maybe simplify this? I.e. I'm not sure you need the next three lines, you could just go straight into the while loop, right?

Reply:
Done.